extensions/wrapper/wrapper-broker-api/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+val sovityEdcClientGroup: String by project
+group = sovityEdcClientGroup
+
+plugins {
+    `java-library`
+    `maven-publish`
+}
+
+dependencies {
+
+    annotationProcessor("org.projectlombok:lombok:1.18.26")
```

Comment:
just to make sure: I know that there were a lot of discussions which dependencies to include and which not. I thought, that especially lombok was not used in core EDC, as IDE and Copilot can easily generate the necessary methods.